assets/css/less/admin-notice-review.less

```diff
@@ -0,0 +1,75 @@
+.wpbdp-review-notice {
+    --primary-color: rgb(65, 153, 253);
```

Comment:
This is an improvement, but the problem is not it doesn't match BD. How about we use the new button styles for BD instead?
https://www.figma.com/file/hey3bHjXZhJpGGt277mHbg/Business_directory?node-id=247%3A598

It would be best if the styles for these buttons go in the admin.less file or something, since they will be used across the admin area.